server/svix-server/src/main.rs

```diff
@@ -115,5 +130,26 @@ async fn main() {
         None => {}
     };
 
+    if let Some(wait_for_seconds) = args.wait_for {
+        let mut wait_for = Vec::with_capacity(2);
+        wait_for.push(wait_for_dsn(
+            &cfg.db_dsn,
+            POSTGRES_NAME,
+            POSTGRES_PORT,
+            wait_for_seconds,
+        ));
+
+        if let Some(redis_dsn) = &cfg.redis_dsn {
+            wait_for.push(wait_for_dsn(
+                redis_dsn,
+                REDIS_NAME,
+                REDIS_PORT,
+                wait_for_seconds,
+            ));
+        }
+
+        futures::future::join_all(wait_for).await;
```

Comment:
Will this actually `panic` the main thread? I think no, as we are doing nothing with the `Result`.

Given my comment from Slack:

> Btw, I just realized, not that it matters, but waiting for both at the same time doesn't do anything, because you have to wait for both anyway, and waiting is instant if it exists.

We probably should just `await.expect()` on both of the `wait_for_dsn` inline. What do you think?